### ISSUE TITLE
Note: GHC 8.6.3 broken on windows for TH

### ISSUE THREAD

**Issue by alanz** [haskell/haskell-ide-engine#1008]
It segfaults.

See https://ghc.haskell.org/trac/ghc/ticket/16057

**Comment by Anrock** [comment]
As a consequnce - hie-8.6.3 can't be build on Windows due to fclabels not being buildable: https://github.com/sebastiaanvisser/fclabels/issues/37

**Comment by alanz** [comment]
Maybe we should adjust the windows build script to skip it by default?

**Comment by Anrock** [comment]
@alanz you mean replace building with some warning in hie-8.6.3 target? Sounds like a good idea.

**Comment by djleonskennedy** [comment]
Hello @alanz will it work on downgraded GHC ?

**Comment by alanz** [comment]
It should work on 8.6.2

**Comment by djleonskennedy** [comment]
@alanz thank you

**Comment by andersyr** [comment]
So that others can avoid the same mistake I made, the [ReadMe](https://github.com/haskell/haskell-ide-engine/blob/master/README.md#install-specific-ghc-version) should use hie-8.6.2 as the **Nightly** installation example and point out that 8.6.3 is currently broken. I.e.,

Install **Nightly** (and hoogle docs):
```
stack ./install.hs hie-8.6.2
stack ./install.hs build-doc-8.6.2
```
(Note, nightly GHC-8.6.3 is currently broken. Please use an alternate version until it's patched.)

**Comment by alanz** [comment]
The nightly is now 8.6.4, and will be used in master when https://github.com/haskell/haskell-ide-engine/pull/1129 lands.

**Comment by jneira** [comment]
Hi, i am getting to install master using stack in windows 10 (with some caveats about cabal-helper-wrapper exe location) but in windows 7 (with stack.yaml and stack-8.6.4.yaml) it fails due to https://github.com/kazu-yamamoto/unix-time/issues/51

**Comment by Anrock** [comment]
`install.hs` no longer tries to build 8.6.3 on Windows, closed by #1168
